tests/security/ima/ima_appraisal_digital_signatures.pm

```diff
@@ -55,7 +55,7 @@ sub run {
 
         # Allow "No such file" message for the files in /proc because they are mutable
         my @finds = split /\n/, $findret;
-        $_ =~ m/\/proc\/.*No such file/ or die "Failed to create security.ima for $_" foreach (@finds);
+        $_ =~ m/\/proc\/.*No such file|evm\/ima signature|hash\(sha256\)|^\w{530}$/ or die "Failed to create security.ima for $_" foreach (@finds);
```

Comment:
What's the meaning of ^\w{530} ?

Reply:
@Amrysliu 
The key value has 530 lens with random digital numbers and characters (in case tested with sha256 hash).
Thanks Xiaojing for this code.